### DISCUSSION TITLE
Is DELPHI dead??? No, more alive than ever before!

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#24] [General]
What a question in the README.md. Is DELPHI dead? No!!! DELPHI is a language with a long tradition, based on Pascal by Nikolas Wirth, made popular with the product Turbo Pascal (programmed by Anders Hejlsberg) and the TUI framework TurboVison, which was added later. Also, the switch was put into the graphical world of Windows. Then came the commercially successful successor DELPHI, also from the "pen" of Anders with the VCL library for Windows programming. This version has been further developed bit by bit until today and there is no end in sight. Many good components and frameworks have been developed for DELPHI over time, e.g. by TMS Software (which are also described and used in this book and the example program). DELPHI has been developed with a sense of proportion into the modern programming by Embarcadero without overloading the language with features (C/C++ and C# are unfortunately "negative" examples). Therefore, it can be said that DELPHI with its vibrant community and companies that develop components and frameworks is looking forward to a bright future that will certainly continue for many decades to come.
So, I come to my statement from the headline... Those who have been declared dead live longer! This has already proven to be true many times in IT and this is especially true for DELPHI. DELPHI will be with us for a long time to come and give us a lot of pleasure in programming applications with the RAD paradigm. So, enjoy reading Holger's book and helping to develop the example application from the book.
So, if someone here is of the opinion that DELPHI is a dead language, we are happy to open a good and constructive discussion here.

**Comment by holgerflick** [comment]
First of all, it's a common joke people make because of exactly what you say. There's even more direct arguments. Microsoft deprecated Winforms in 2005, and also told us there would be no more native code on Windows within about 5 years. Well, go figure, it never happened. I still also see dozens of applications written in Winforms each year.
Second, I added your conclusion to the readme.
Third, yes, we do indeed welcome any discussion on this topic.
